### DISCUSSION TITLE
Warning when upgrading FOSS repo packages (zimbra-nginx fail to bind privileged ports)

### DISCUSSION THREAD

**Discussion by jeredfloyd** [ianw1974/zimbra-build-scripts#76] [Show and tell]
If you are running a Zimbra 10 FOSS build, you are still pulling the open source components from the main Zimbra repo but separately installing the Zimbra components, so expected upgrade actions may not occur when you upgrade FOSS components.

I just encountered this after upgrading from zimbra-nginx-1.20.0-1zimbra8.8b4.el8.x86_64 to zimbra-nginx-1.24.0-1zimbra8.8b4.el8.x86_64 as part of my system package manager update.  (This is considered part of the 10.0.7 patch release.)  Nginx (zimbra-proxy) failed to start as it could not bind privileged ports.

If you run into this, as root you should run either:
`  /opt/zimbra/libexec/zmfixperms`
or simply:
`  setcap CAP_NET_BIND_SERVICE=+ep /opt/zimbra/common/sbin/nginx`

**Comment by ianw1974** [comment]
I had something similar a short while back when I also upgraded packages on my Ubuntu install of Zimbra.  I also ran ```zmfixperms``` which resolved the problem.
